hack/test-manifests-openstack.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
```

Comment:
Do you want to put this file at `hack/openstack/test-manifests.sh` so that the OpenStack team can own it?

Reply:
Thank you, great suggestion. Done!